spec/integration_spec.rb

```diff
@@ -1,3 +1,4 @@
+require 'byebug'
```

Comment:
Ups, I forgot to remove it. It's already specified in the gemspec it just needs to be autorequired, as done in https://github.com/redbooth/percona_migrator/commit/9afc111da6126e8b430c7405b70b636006c15e9f already.